curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f1b6e60bd9edf902c81fd.md

```diff
@@ -43,7 +43,7 @@ const count = 8;
 const rows = [];
 
 --fcc-editable-region--
-for (let i = 0; i < count; "iteration") {
+for (let i = 0; i < count; i = i + 1;) {
```

Comment:
no, this doesn't solve the issue. Please read the issue carefully.

Reply:
It would be much helpful for me if you can describe the issue a bit more?